### DISCUSSION TITLE
Can't update Panels

### DISCUSSION THREAD

**Discussion by akj501** [Blackymas/NSPanel_HA_Blueprint#2057] [Q&A]
Hi All,

Within home assistant my nspanels show up under the ESPHome tab and say they need to be updated., however when I click update I get the following error:

INFO ESPHome 2024.3.2
INFO Reading configuration /config/esphome/masterbedroompanel.yaml...
ERROR Error while reading config: Invalid YAML syntax:

could not determine a constructor for the tag '!input'
  in "/config/esphome/packages/nspanel_esphome.yaml", line 3769, column 21
  
 Any help would be much appreciated :)

**Comment by edwardtfn** [comment]
Hej @akj501, sorry for the late reply here.
Are you still facing this issue?

**Reply by akj501** [reply]
Hi @edwardtfn - sorry for the delay in coming back to you.

Yes unfortunately still having issues updating.

**Reply by edwardtfn** [reply]
Do you mind reporting this as a bug?
https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/new/choose
